main.py

```diff
@@ -2,6 +1,48 @@
+import init_django_orm # noqa:F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
-def main() -> QuerySet:
-    pass
+def main():
+    # Greate genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Drama")
+
+    # Create actors and actresses
+    Actor.objects.create(first_name="George",
+                         last_name="Klooney")
+    Actor.objects.create(first_name="Kianu",
+                         last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett",
+                         last_name="Keegan")
+    Actor.objects.create(first_name="Will",
+                         last_name="Smith")
+    Actor.objects.create(first_name="Jaden",
+                         last_name="Smith")
+    Actor.objects.create(first_name="Scarlett",
+                         last_name="Johansson")
+
+    # Update operations
+    (Genre.objects.filter(name="Dramma")
+     .update(name="Drama"))
+    (Actor.objects.filter(last_name="Klooney")
+     .update(last_name="Clooney"))
+    (Actor.objects.filter(first_name="Keanu")
+     .update(last_name="Reeves"))
+
+    # Delete operations
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
```

Comment:
Deleting all actors with the first name 'Scarlett' might not be intended, as it will remove both 'Scarlett Keegan' and 'Scarlett Johansson'. Ensure this is the desired behavior.